src/app/core/services/rundown-timing-context-state.service.ts

```diff
@@ -0,0 +1,187 @@
+import { Rundown } from '../models/rundown'
+import { BehaviorSubject, Observable, Subscription, SubscriptionLike } from 'rxjs'
+import { RundownStateService } from './rundown-state.service'
+import { RundownTimingService } from './rundown-timing.service'
+import { RundownTimingContext } from '../models/rundown-timing-context'
+import { Injectable } from '@angular/core'
+import { Segment } from '../models/segment'
+
+const HIGH_RESOLUTION_INTERVAL_DURATION_IN_MS: number = Math.floor(1000 / 30)
+const LOW_RESOLUTION_INTERVAL_DURATION_IN_MS: number = Math.floor(1000 / 4)
+
+@Injectable()
+export class RundownTimingContextStateService {
+  private timeResolutionTimerId?: ReturnType<typeof setTimeout>
+  private isHighResolutionTimer: boolean = false
```

Comment:
This attribute feels unnecesary. Wouldn't it be enough to just check whether there is an active Rundown or not?

Reply:
See https://github.com/tv2/ng-sofie/pull/67#discussion_r1401972906.